proxy.yml

```diff
@@ -0,0 +1,16 @@
+config:
+  target: 'http://localhost:3009'
+  # http:
+  #   pool: 20
+  phases:
+    - duration: 180
+      arrivalRate: 10
+  defaults:
+    headers:
+      x-my-service-auth: '987401838271002188298567'
+scenarios:
+  - flow:
+    - loop:
+      - get:
+          url: "/restaurants/voluptatem9999975/menu/lunch"
```

Comment:
I noticed this tests the same endpoint repeatedly. Might be best to get some different values in her to make your tests more realistic.